test/metabase/query_processor_test/filter_test.clj

```diff
@@ -151,8 +151,9 @@
              (mt/first-row
                (mt/format-rows-by [int]
                  (mt/run-mbql-query checkins
-                   {:aggregation [[:count]]
-                    :filter      [:not-null *date]}))))))))
+                   {:source-query {:source-table $$checkins}
+                    :aggregation  [[:count]]
+                    :filter       [:not-null *date]}))))))))
 
```

Comment:
Fixed because you weren't really supposed be using field literals without a source query in the first place. `*date` is shorthand for `[:field (:name (Field (mt/id :checkins :date)) {:base-type (:base_type (Field (mt/id :checkins :date)))}]` in this case